src/components/simulator/Simulator.tsx

```diff
@@ -1061,9 +1062,7 @@ export class Simulator extends React.Component<SimulatorProps, SimulatorState> {
         <div className={styles.simulator_tab + ' ' + tabHidden} onClick={this.onToggle}>
           <div className={styles.simulator_tab_icon + ' fe-smartphone'} />
           <div className={styles.simulator_tab_text}>
-            Run in
-            <br />
```

Comment:
checked that we get same wrapping without this
<img width="144" alt="Captura de Pantalla 2021-03-04 a la(s) 11 46 38" src="https://user-images.githubusercontent.com/675558/110000601-807b9280-7ce1-11eb-8d70-a05bedfcddb5.png">